### PR TITLE
Disable user select on tablets, Fixes #2655

### LINKED ISSUE #2655: Disable user select on tablets
We have had to do this on two of our flagship client apps. When a user holds down their finger to bring up the context menu the text on the page gets selected and it's not a good look. Is there any reason NOT to do this at the Hoist level?

### PR DESCRIPTION
**Pull request by febbraiod**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [X] Caught up with `develop` branch as of last change.
- [X] Added CHANGELOG entry, or determined not required.
- [X] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [X] Updated doc comments / prop-types, or determined not required.
- [X] Reviewed and tested on Mobile, or determined not required.
- [X] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by febbraiod** [comment]
I suppose this _could_ be a breaking change for an app that relies on user select on tablets, but that seems very unlikely, so I decided that we didn't really need a CL entry. Correct me if wrong.

File: styles/XH.scss [hunk 1]
```diff
@@ -194,3 +194,7 @@ body.xh-app {
     text-transform: uppercase;
   }
 }
+
+.xh-tablet * {
+  user-select: none !important;
+}
```
Comment (amcclain) [inline styles/XH.scss:200]: I would nest (like the other rules within this file) underneath the `body.xh-app` selector. Not sure if that would make a functional difference but still seems correct and consistent.  (So, move up inside braces and add leading ampersand: `&.xh-tablet * {...}`

Is the `!important` required?

**Comment by amcclain** [comment]
> I suppose this _could_ be a breaking change for an app that relies on user select on tablets, but that seems very unlikely, so I decided that we didn't really need a CL entry. Correct me if wrong.

I agree, I do not think we need a CL entry here.

File: styles/XH.scss (hunk 1 above)
Comment (febbraiod) [reply styles/XH.scss:200]: Will move to that block now. 

The !important comes from the definition in the client app from which this change originated. I'm not sure it's needed, but someone thought it was. Could test in MY dashboard related client app but would need to cut a release and hit the app with my tablet via blackberry access to be sure (which is fine but will take some time)

**Comment by amcclain** [comment]
👍 thanks for getting this in

**Pull request merged**